README.md

```diff
@@ -348,57 +348,25 @@ import smile.identity.core.IDParameters;
 import smile.identity.core.IDApi;
 ```
 
-##### submit_job method
+##### submitJob method
 
 Your call to the library will be similar to the below code snippet:
 ```java
-  PartnerParameters partnerParameters = new PartnerParameters(<String user_id>, <String job_id>, <Integer 5>);
+  PartnerParams partnerParams = new PartnerParams(<JobType jobType>, <Map<String, Object> optionalInfo>, <String userId>, <String jobId>);
```

Comment:
Just a suggestion, it'll are great to have all the required fields come first before the optional one

Reply:
@ayinloya Great suggestion! I will make this change, since it will also require me updating the partnerParameters class, I will include this change in the webapi pr ( which is next up 👍🏾 )